packages/ui/feature-builder-test-steps/src/lib/test-action/test-action.component.ts

```diff
@@ -247,3 +249,16 @@ export class TestActionComponent extends TestStepCoreComponent {
       );
   }
 }
+
+function formatErrorMessage(errorMessage: string): string {
+  const errorMessagesSplit = errorMessage.split('Error:');
+  if (errorMessagesSplit.length < 2) {
+    return errorMessage;
+  }
+
+  const indentationStep = '  ';
+  return errorMessagesSplit.reduce((acc, current, index) => {
+    const indentation = indentationStep.repeat(index);
+    return `${acc}${indentation}Error ${index}: ${current.trim()}\n`;
```

Comment:
make this:
`return `${acc}${indentation}Error ${index+1}: ${current.trim()}\n`;`